scripts/GetBlockData.ts

```diff
@@ -0,0 +1,23 @@
+// Run: npx hardhat run scripts/GetBlockData.ts --network childTest
```

Comment:
It will be helpful to add this (and the other scripts) in the `package.json`, in my opinion. For example: `"SumBlocksForAll:root": "npx hardhat run ./scripts/SumBlocksForAll.ts --network root"`, so, one will be able to just run `npm run SumBlocksForAll` and the script gonna run.

Reply:
added scripts that runs with npm run ScriptName